src/qml/pages/wallet/CoinSelection.qml

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2025 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import org.bitcoincore.qt 1.0
+
+import "../../controls"
+import "../../components"
+
+Page {
+    id: root
+
+    property WalletQmlModel wallet: walletController.selectedWallet
+
+    signal done()
+
+    header: NavigationBar2 {
+        centerItem: Header {
+            headerBold: true
+            headerSize: 18
+            header: qsTr("Coin Selection")
+        }
+        rightItem: NavButton {
+            text: qsTr("Done")
+            onClicked: root.done()
+        }
+    }
+
+    background: Rectangle {
+        color: Theme.color.neutral0
+    }
+
+    ColumnLayout {
+        id: header
+        anchors.top: parent.top
+        anchors.horizontalCenter: parent.horizontalCenter
+        width: Math.min(parent.width - 40, 450)
+
+        RowLayout {
+            Layout.fillWidth: true
+            spacing: 15
+            CoreText {
+                Layout.alignment: Qt.AlignLeft
+                Layout.fillWidth: true
+                Layout.preferredWidth: 0
+                font.pixelSize: 18
+                color: Theme.color.neutral9
+                elide: Text.ElideMiddle
+                wrapMode: Text.NoWrap
+                horizontalAlignment: Text.AlignLeft
+                text: qsTr("Total selected")
+            }
+            CoreText {
+                Layout.alignment: Qt.AlignRight
+                color: Theme.color.neutral9
+                font.pixelSize: 18
+                text: root.wallet.coinsListModel.totalSelected
+            }
+        }
+        RowLayout {
+            Layout.bottomMargin: 30
+            CoreText {
+                Layout.alignment: Qt.AlignLeft
+                Layout.fillWidth: true
+                Layout.preferredWidth: 0
+                font.pixelSize: 15
+                color: Theme.color.neutral7
+                elide: Text.ElideMiddle
+                wrapMode: Text.NoWrap
+                horizontalAlignment: Text.AlignLeft
+                text: if (root.wallet.coinsListModel.overRequiredAmount) {
+                    qsTr("Over required amount")
+                } else {
+                    qsTr("Remaining to select")
```

Comment:
nit, if exact amount is selected is it good UX to have _Remaining to select 0.000000_?